sympy/ntheory/factor_.py

```diff
@@ -2201,3 +2201,28 @@ def is_deficient(n):
     if is_perfect(n):
         return False
     return bool(abundance(n) < 0)
+
+
+def is_amicable(m, n):
+    """Returns True if the pair `(m, n)` is a Amicable Number, else False.
+
+    Amicable numbers are two different numbers so related that the sum
+    of the proper divisors of each is equal to the other number.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import divisor_sigma, is_amicable
```

Comment:
don't import `divisor_sigma` here in the docstring and change 'a' to 'an' in first sentence.